### ISSUE TITLE
Incorrect typings path

### ISSUE THREAD

**Issue by pietro-marrone** [yorickshan/html2canvas-pro#15]
Hi!
Thanks for taking the time to maintain this library!

I noticed that the types couldn't be used in my project: attempting to import any of the exported obejcts (html2canvas, Options), the typescript compiler emits this error:
```
Could not find a declaration file for module 'html2canvas-pro'. '/Users/pmarrone/Dev/measurement/attribution-fragset/node_modules/html2canvas-pro/dist/html2canvas-pro.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/html2canvas-pro` if it exists or add a new declaration (.d.ts) file containing `declare module 'html2canvas-pro';`
``` 

Looking at the dist folder, it seems that the path set in package json is wrong, since the types are actually defined inside
```
dist/types/src/index.d.ts
```
instead of 
```
dist/types/index.d.ts
```

Updating the path in my local dist folder makes the types of the lib recognized by my project. Could you please take a look?

Thanks a lot!

**Comment by yorickshan** [comment]
Hi there, thank you for your feedback. This bug has been resolved in html2canvas-pro version 1.5.5.

**Comment by yorickshan** [comment]
If you encounter any issues with html2canvas-pro or have suggestions for new features, feel free to submit a PR. I'll address it as soon as possible.
